src/en/caas.md

```diff
@@ -0,0 +1,192 @@
+Title: CAAS and Juju
+TODO:  Once 2.4 is officially released remove support Note and update status output
+       Should eventually link to CAAS-charm developer documentation
+       Update when storage becomes a Juju drivable aspect.
+       Add architectural overview/diagram once CAAS becomes stable.
+       Consider manually adding a cluster (third-party installs) via `add-cloud` and `add-credential`
+
+# CAAS and Juju
+
+CAAS is *Containers as a Service*, a cloud service that allows users to upload,
```

Comment:
Given the Kubernetes focus, we probably just want to say something like that Juju supports managing workloads on Kubernetes in a similar way to how it manages workloads on VM based clouds etc etc